src/components/panel/panel.js

```diff
@@ -865,15 +875,15 @@ MdPanelRef.prototype.close = function() {
         return self.detach();
       }, this._$q.reject)
       .then(function() {
-        // TODO(ErinCoughlan) - Add destroy. This will make the code here
-        // different than just calling this.detach().
+        return self.destroy();
```

Comment:
You're still calling destroy as part of close. I thought we wanted close to hide and detach, but not make it impossible to use again.